types.d.ts

```diff
@@ -7197,7 +7220,7 @@ declare abstract class MultiWatching {
 	invalidate(callback?: any): void;
 	suspend(): void;
 	resume(): void;
-	close(callback: CallbackFunction<void>): void;
+	close(callback: CallbackFunction_1<void>): void;
```

Comment:
You missed a few places:

* MultiWatching.close
* Watching.close
* Watching.invalidate
* Watching.handler
* Watching.callbacks
* AsyncQueue.add
* AsyncQueue.waitFor

Reply:
I commit ([0105d7d](https://github.com/luffy84217/webpack/commit/0105d7d3e6a2d1a0bb18b7cd9fdf83ec78121cc7)) to make up omitted callback types you listed